aiounifi/models/device.py

```diff
@@ -338,6 +338,20 @@ class TypedDeviceWlanOverrides(TypedDict):
     wlan_id: str
 
 
+class TypedDeviceSpeedtestStatus(TypedDict):
+    """Device speedtest status type definition."""
+
+    latency: int
+    rundate: int
+    runtime: int
+    status_download: int
+    status_ping: int
+    status_summary: int
+    status_upload: int
+    xput_download: float
+    xput_upload: float
```

Comment:
I just have one question, is this still the same on the latest versions of the network app?

Reply:
Yes, for me it is (UDM SE, Network  v8.0.26).

Reply:
Awesome!